.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+name: CI
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [master]
+jobs:
+  test:
+    runs-on:  macos-latest
+    steps:
+    - name: Clone Repo
+      uses: actions/checkout@v1
+    - name: Gem Cache
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gem-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gem-
+    - name: Cocoapod Cache
+      uses: actions/cache@v1
+      with:
+        path: Pods
+        key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-pods-
+    - name: Bundle Install
+      run: bundle --path vendor/bundle
+    - name: Pod Install
+      run: |
+        bundle exec pod install
+    - name: Run Tests
+      run: |
+        bundle exec fastlane test scheme:"UnitTests"
+        bundle exec fastlane test scheme:"IntegrationTests"
```

Let's separate the unit tests from the integration test step so it's easier to grok later if something goes wrong

```suggestion
    - name: Run Unit Tests
      run: bundle exec fastlane test scheme:"UnitTests"
    - name: Run Integration Tests
      run: bundle exec fastlane test scheme:"IntegrationTests"
```

---

I would recommend the same